hack/lib.sh

```diff
@@ -39,7 +39,7 @@ KUBEBUILDER_VERSION=2.3.2
 KUSTOMIZE_BIN=$OUTPUT_BIN/kustomize
 KUSTOMIZE_VERSION=3.10.0
 KUBETEST2_VERSION=v0.1.0
-KUBETSTS2_BIN=$OUTPUT_BIN/kubetest2
+export KUBETSTS2_BIN=$OUTPUT_BIN/kubetest2
```

Comment:
This var doesn't need to be exported, as it will be loaded at:

https://github.com/chaos-mesh/chaos-mesh/blob/42fcabc19f5268cf0e0808d6ea7f2171fba2f28a/hack/e2e.sh#L28